spec.html

```diff
@@ -22544,16 +22544,15 @@
 
       <!-- es6num="19.1.2.2" -->
       <emu-clause id="sec-object.create">
-        <h1>Object.create ( _O_ [ , _Properties_ ] )</h1>
+        <h1>Object.create ( _O_, _Properties_ )</h1>
```

Comment:
The `Properties` argument is conceptually optional - the function works without it. Why should it be made to be required? I'd strongly prefer the deviating explicit length to altering the meaning of the arguments :-/